internal/integration/provision/upgrade.go

```diff
@@ -285,7 +291,7 @@ func init() {
 		&UpgradeSuite{specGen: upgradePreviousToStable, track: 0},
 		&UpgradeSuite{specGen: upgradeStableToCurrent, track: 1},
 		&UpgradeSuite{specGen: upgradeCurrentToCurrent, track: 2},
-		&UpgradeSuite{specGen: upgradeStableToCurrentPreserve, track: 0},
+		&UpgradeSuite{specGen: upgradeCurrentToCurrentBios, track: 2},
```

Comment:
```suggestion
		&UpgradeSuite{specGen: upgradeCurrentToCurrentBios, track: 0},
```

I would probably keep it in the same track, as track 2 runs all Kubernetes versions, which is already 40 minutes long.